lib/transport/data_validation/repository/feed_version_repository.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Transport.DataValidation.Repository.FeedVersionRepository do
+  @moduledoc """
+  A feed version repository to interact with datatools.
+  """
+
+  alias Transport.DataValidation.Aggregates.FeedVersion
+  alias Transport.DataValidation.Queries.FindFeedVersion
+
+  @endpoint Application.get_env(:transport, :datatools_url) <> "/api/manager/secure/feedversion"
+  @client HTTPoison
+  @res HTTPoison.Response
+  @err HTTPoison.Error
+
+  @doc """
+  Finds a feed version by its id.
+  """
+  @spec execute(FindFeedVersion.t) :: {:ok, FeedVersion.t} | {:ok, nil} | {:error, any()}
+  def execute(%FindFeedVersion{project: %{id: project_id}, latest_version_id: latest_version_id}) when is_binary(project_id) and is_binary(latest_version_id) do
```

Comment:
Check at https://github.com/elixir-lang/elixir/blob/master/lib/logger/lib/logger/translator.ex for styling of long function declarations.